src/wwwroot/js/genpage/gentab/presets.js

```diff
@@ -10,6 +10,45 @@ class PresetHelpers {
     }
 }
 
+/** Manages preset links for models and LoRAs. User-scoped storage in User.UserUIData.ItemPresetLinks */
```

Comment:
`User-scoped storage in User.UserUIData.ItemPresetLinks` this is just... strange text to put in a js class doc.